pyproject.toml

```
@@ -24,7 +24,7 @@ dependencies = [
     "voila>=0.4,<0.5",
     "pyyaml>=5.4.1",
     "specutils>=1.15",
-    "specreduce>=1.3.0,<1.4.0",
+    "specreduce>=1.4.0",
```

Should we exclude 1.5 now since we're allowing breaking changes still in specreduce?

```suggestion
    "specreduce==1.4.*",
```

---

i think we need this pinned to 1.4.1 or else this feature wont work?